README.md

```diff
@@ -150,8 +150,33 @@ Please use GitHub’s issue tracker and pull request features.
 
 2. Install: `npm install`
 
-3. Use `npm run serve` to launch a local instance of the service. This will load
-   data from Cloud Storage; data can take up to a minute to be loaded and ready.
+3. Launch the local server:
+
+```zsh
+mkdir -p data/json
+```
+
+Then, either:
+
+   1. Copy the latest data:
+
+```zsh
+mkdir -p data/json
+cd data/json
+curl https://storage.googleapis.com/web-api-confluence-data-cache/latest/json/org.chromium.apis.web.ApiVelocityData.json > org.chromium.apis.web.ApiVelocityData.json
```

Comment:
Confirmed that these commands work for me.

Reply:
:+1: